roles/pulibrary.svn/tasks/main.yml

```diff
@@ -1,2 +1,6 @@
 ---
 # tasks file for roles/pulibrary.svn
+- name: add mod svn
+  apt:
+    name: ["libapache2-mod-svn"]
```

Comment:
Aren't we using `svnserve` instead of `mod_dav_svn`?

Reply:
indeed. was "cargo culting" this documentation.

Reply:
👍